sanity/schemas/components/imageSource.js

```diff
@@ -0,0 +1,27 @@
+export default {
+  name: 'imageSource',
+  title: 'Zdjęcie ze źródłem',
+  type: 'object',
+  fields: [
+    {
+      title: 'Zdjęcie',
+      name: 'image',
+      type: 'image',
+    },
+    {
+      title: 'Źródło',
+      name: 'source',
+      type: 'markdown',
```

Comment:
Change the type and title globally for that field

Reply:
same as above